docs/features/techdocs/extensions.md

```diff
@@ -0,0 +1,54 @@
+---
+id: extensions
+title: Using TechDocs Extensions
+sidebar_label: Using TechDocs Extensions
+description: How to use the built-in TechDocs extension points
+---
+
+# TechDocs Backend Extensions
+
+The TechDocs backend plugin provides the following extension points:
+
+- `techdocsBuildsExtensionPoint`
+- `techdocsGeneratorExtensionPoint`
+- `techdocsPreparerExtensionPoint`
+- `techdocsPublisherExtensionPoint`
+
+## Build Extensions
+
+### Set Log Transport
+
+By default, the TechDocs build phase does not log out much information while building out TechDocs. However, the
+`techdocsBuildsExtensionPoint` can be used to setup a custom Winston transport for TechDocs build logs.
+
+Here is an example of logging to console:
+
+```typescript jsx title="packages/backend/src/extensions/techDocsExtension.ts"
+import { techdocsBuildsExtensionPoint } from '@backstage/plugin-techdocs-backend';
+import { createBackendModule } from '@backstage/backend-plugin-api';
+import { transports } from 'winston';
+
+export const techDocsExtension = createBackendModule({
+  pluginId: 'techdocs',
+  moduleId: 'extension',
```

Comment:
I recommend using a module id that indicates what the extension does, like `console-build-transport-log`

Reply:
Same for the variable name, I would suggest renaming to `techdocsConsoleBuildTransportLogModule` instead. Wdyt?

Reply:
Addressed!